app/services/file_manager.rb

```diff
@@ -49,7 +49,7 @@ def mime_type
   end
 
   def upload
-    uploader.upload
+    uploader.upload(file: encrypted_file)
   end
```

Comment:
As discussed below, we could just pass the encrypted data blob here, it saves new upping a `Tempfile`/`File` object that gets `.read` called on it anyway.